webservices/templates/swagger-ui.html

```diff
@@ -23,9 +23,45 @@
 
       body
       {
-        margin:0;
-        background: #fafafa;
+      margin:0;
```

Comment:
Just a FLAK8 formatting issue: there should be a space before `0`

Reply:
fixed!  Thanks @johnnyporkchops